mmcv/cnn/bricks/registry.py

```diff
@@ -1,13 +1,54 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+import torch.nn as nn
+
 from mmcv.utils import Registry
 
+
+def build_upsample_layer_from_cfg(cfg, registry, *args, **kwargs):
+    """Build upsample layer.
+
```

Comment:
Add docstring to indicates what is the difference between build_upsample_layer_from_cfg and build_from_cfg

Reply:
Since the interfaces of all UPSAMPLE_LAYERs modules are different.
I have added usage and deprecate information.

Reply:
I just wrap `build_upsample_layer` as a build_func (`build_upsample_layer_from_cfg`) and `build_upsample_layer` can be deprecated in the future.